### DISCUSSION TITLE
Edit metadata

### DISCUSSION THREAD

**Discussion by logicchild0x00** [icsharpcode/ILSpy#2957] [Development] [answered]
I just wonder if can you edit the metadata of a DLL file?
Example: Under Metadata -> Tables -> TypeRef, I need to edit the ResolutionScope value of an entry.
I was able doing this using dnSpy with no issue at all, and it was very helpful.

ILSpy:
![ILSpy](https://user-images.githubusercontent.com/42699587/230790951-ebccda3e-a047-41ac-bf29-4369357388eb.png)

dnSpy:
![dnSpy](https://user-images.githubusercontent.com/42699587/230790958-94b3d19d-c0e7-46fc-a2be-86a6225aa75a.png)

**Comment by christophwille** [comment, marked as answer]
Modifying assemblies is not part of the feature set of ILSpy. Our main focus is the quality of the decompilation engine (and eg dnSpyEx relies on our engine).

**Reply by logicchild0x00** [reply]
I can see now, thanks.
